bundles/org.eclipse.ui.themes/css/e4_default_gtk.css

```diff
@@ -190,7 +190,7 @@ CTabFolder Canvas {
 }
 
 .MPartStack{
-	swt-selected-tab-highlight: #a0a0a0;
+	swt-selected-tab-highlight: #8a8a8a;
```

Comment:
But in https://github.com/eclipse-platform/eclipse.platform.ui/pull/2455/files#diff-c17f7415d0f31b4e2531cc20f515b87c405ce80099aa4730282ce6d27a59d0c8

I did not change this color: I did only change it from #A0A0A0 to #a0a0a0

Can you explain pls? What is bug already there before #2455 and not a regression of it?